com.unity.shadergraph/Editor/GraphDeltaRegistry/Registry/RegistryPlaceholder.cs

```diff
@@ -136,6 +141,64 @@ public ShaderFoundry.ShaderFunction GetShaderFunction(INodeReader data, ShaderFo
             }
         }
 
+        // TODO (Brett) [BEFORE RELEASE] The doc in this class is a basic 
+        // description of what needs to be done to add nodes.
+        // CLEAN THIS DOC
```

Comment:
Just making sure this todo is a future one

Reply:
Yeah, I'm just looking for a way to remind myself that something needs to happen at some point. If you have a preferred way, just let me know.

Reply:
I'll add a task in jira for us to clean code comments and code doc in the March sprint.